database/migrations/2017_12_21_031258_create_roles_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRolesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('roles', function (Blueprint $table) {
+            $table->increments('id');
+            $table->enum('type', ['user', 'admin']);
+            $table->timestamps();
```

Comment:
We can delete timestamps since we are not going to change roles.

To make this work, we need to add the `timestamps` property in the Role class and set it to false. 
See subsection `Timestamps` in https://laravel.com/docs/5.5/eloquent#eloquent-model-conventions